### PR TITLE
Remove superfluous #nullable enables

### PR DESCRIPTION
**Pull request by stephentoub**
cc: @jeffhandley, @buyaa-n, @krwq

### CODE REVIEW

**Review by eiriktsarpalis — APPROVED** [review]
LGTM if CI is green.

**Comment by ViktorHofer** [comment]
Just curious, how did you apply the change? Via automation or manually?

**Comment by stephentoub** [comment]
Regex replace in VS, and then reverting files where it's still needed.  Build -allconfigurations passed locally but apparently I need to revert another file to make CI happy.

**Review by safern — APPROVED** [review]

File: src/libraries/Directory.Build.props [hunk 1]
```diff
@@ -51,6 +51,8 @@
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true'">
     <!-- don't warn on usage of BinaryFormatter from test projects -->
     <NoWarn>$(NoWarn);SYSLIB0011</NoWarn>
+    <!-- allow nullable annotated files to be incorporated into tests without warning -->
+    <Nullable Condition="'$(Nullable)' == '' and '$(Language)' == 'C#'">annotations</Nullable>
```
Comment (buyaa-n) [inline src/libraries/Directory.Build.props:55]: Not sure how, but seems test failures related to this change

**Review by buyaa-n — APPROVED** [review]
Looks good to me, but test failures might be related to the change

File: src/libraries/Directory.Build.props (hunk 1 above)
Comment (stephentoub) [reply src/libraries/Directory.Build.props:55]: They are.  There are now some attributes on a method that was explicitly testing that there weren't any attributes :) Ugh.

**Pull request merged**